homeassistant/components/minecraft_server/sensor.py

```diff
@@ -75,11 +75,11 @@ def available(self) -> bool:
 class MinecraftServerVersionSensor(MinecraftServerSensorEntity):
     """Representation of a Minecraft Server version sensor."""
 
-    _attr_translation_key = "version"
+    _attr_translation_key = KEY_VERSION
```

Comment:
There is a lot of cleanups in this PR as well as "fixing the issue".
Can we move the cleanups to a separate PR first and then rebase this on top so it becomes easier to review.

Reply:
Sorry, I still have to get used to create more smaller PRs... I'm on it, thanks.